### PR TITLE
fix: new protocol and chain token list

### PR DESCRIPTION
**Pull request by LuPoYi**

### CODE REVIEW

File: packages/lending/src/adapter.config.ts [hunk 1]
```diff
@@ -39,13 +48,24 @@ export const configs: Config[] = [
...
     primaryStablecoin: metisTokens['m.USDC'],
     secondaryStablecoin: metisTokens['m.DAI'],
     primaryNonstablecoin: metisTokens.WETH,
   },
+  {
+    chainId: common.ChainId.base,
+    primaryStablecoin: baseTokens.DAI,
+    secondaryStablecoin: baseTokens.USDC,
+    primaryNonstablecoin: baseTokens.WETH,
+  },
```
Comment (zodahu) [inline packages/lending/src/adapter.config.ts]: Why is `DAI` primary token on Base?

File: packages/lending/src/adapter.config.ts [hunk 2]
```diff
@@ -39,13 +48,24 @@ export const configs: Config[] = [
...
+  {
+    chainId: common.ChainId.gnosis,
+    primaryStablecoin: gnosisTokens.USDC,
+    secondaryStablecoin: gnosisTokens.USDT,
+    primaryNonstablecoin: gnosisTokens.WETH,
+  },
```
Comment (zodahu) [inline packages/lending/src/adapter.config.ts]: Seems `WXDAI` is most commonly stablecoins on Gnosis chain, do we want to use it?

File: packages/lending/src/tokens/gnosis.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "decimals": 18,
+    "symbol": "WXDAI",
+    "name": "Wrapped xDAI"
+  },
+  "USDC": {
+    "chainId": 100,
+    "address": "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
+    "decimals": 6,
+    "symbol": "USDC",
+    "name": "USDC on xdai"
```
Comment (Jeff-CCH) [inline packages/lending/src/tokens/gnosis.json]:
```suggestion
    "name": "USD//C on xDAI"
```

File: packages/lending/src/tokens/gnosis.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "USDT": {
+    "chainId": 100,
+    "address": "0x4ECaBa5870353805a9F068101A40E0f32ed605C6",
+    "decimals": 6,
+    "symbol": "USDT",
+    "name": "USDT on xdai"
```
Comment (Jeff-CCH) [inline packages/lending/src/tokens/gnosis.json]:
```suggestion
    "name": "Tether USD on xDai"
```

File: packages/lending/src/tokens/gnosis.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "WETH": {
+    "chainId": 100,
+    "address": "0x6A023CCd1ff6F2045C3309768eAd9E68F978f6e1",
+    "decimals": 18,
+    "symbol": "WETH",
+    "name": "Wrapped Ether on xdai"
```
Comment (Jeff-CCH) [inline packages/lending/src/tokens/gnosis.json]:
```suggestion
    "name": "Wrapped Ether on xDai"
```

File: packages/lending/src/tokens/gnosis.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    "decimals": 18,
+    "symbol": "XDAI",
+    "name": "xDAI"
+  },
+  "WXDAI": {
+    "chainId": 100,
+    "address": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
+    "decimals": 18,
+    "symbol": "WXDAI",
+    "name": "Wrapped xDAI"
```
Comment (Jeff-CCH) [inline packages/lending/src/tokens/gnosis.json]:
```suggestion
    "name": "Wrapped XDAI"
```

File: packages/lending/src/adapter.config.ts (hunk 1 above)
Comment (LuPoYi) [reply packages/lending/src/adapter.config.ts]: updated to most common stable token.

File: packages/lending/src/adapter.config.ts (hunk 2 above)
Comment (LuPoYi) [reply packages/lending/src/adapter.config.ts]: fixed

File: packages/lending/src/tokens/gnosis.json (hunk 3 above)
Comment (LuPoYi) [reply packages/lending/src/tokens/gnosis.json]: fixed

File: packages/lending/src/tokens/gnosis.json (hunk 4 above)
Comment (LuPoYi) [reply packages/lending/src/tokens/gnosis.json]: fixed

File: packages/lending/src/tokens/gnosis.json (hunk 6 above)
Comment (LuPoYi) [reply packages/lending/src/tokens/gnosis.json]: fixed

File: packages/lending/src/tokens/base.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    "symbol": "USDC",
+    "name": "USD Coin"
+  },
+  "DAI": {
+    "chainId": 8453,
+    "address": "0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb",
+    "decimals": 18,
+    "symbol": "DAI",
+    "name": "Dai Stablecoin"
+  },
```
Comment (zodahu) [inline packages/lending/src/tokens/base.json]: Remove DAI as there is no usage of `baseTokens.DAI`

File: packages/lending/src/tokens/gnosis.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    "symbol": "USDC",
+    "name": "USD//C on xDAI"
+  },
+  "USDT": {
+    "chainId": 100,
+    "address": "0x4ECaBa5870353805a9F068101A40E0f32ed605C6",
+    "decimals": 6,
+    "symbol": "USDT",
+    "name": "Tether USD on xDai"
+  },
```
Comment (zodahu) [inline packages/lending/src/tokens/gnosis.json]: Remove USDT as there is no usage of `gnosisTokens.USDT`

**Review by Jeff-CCH — APPROVED** [review]
LGTM

File: packages/lending/src/tokens/base.json (hunk 7 above)
Comment (LuPoYi) [reply packages/lending/src/tokens/base.json]: removed

File: packages/lending/src/tokens/gnosis.json (hunk 8 above)
Comment (LuPoYi) [reply packages/lending/src/tokens/gnosis.json]: removed

**Comment by zodahu** [comment]
LGTM

**Pull request merged**